amundsen_application/static/js/components/SearchPage/SearchBar/index.tsx

```diff
@@ -61,11 +67,7 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
     const searchTerm = this.state.searchTerm.trim();
     event.preventDefault();
     if (this.isFormValid(searchTerm)) {
-      let pathName = '/';
-      if (searchTerm !== '') {
-        pathName = `/search?searchTerm=${searchTerm}`;
-      }
-      this.props.history.push(pathName);    
+      this.props.submitSearch(searchTerm);
```

Comment:
Notable change: submitting an empty search will no longer navigate to the homepage. I checked Facebook, Pinterest, Google, Gmail, and YouTube. None of these apps let you submit an empty string. Going home is done by clicking on the logo in the navbar.

I'll add some code in `isFormValid` to prevent empty string searches.

Reply:
Good observation in regards to checking other standards. I can't remember if that feature was ever an intentional decision or if I had just assumed that what users wanted. How about we raise this change when we run future updates past Product and Design for QA so that we don't get bugs filed for it.

Reply:
I think it was merely a side effect of how the old `SearchPage` worked. Before the `HomePage` was split out, the `SearchPage` logic was to show `PopularTables` when the `searchTerm` was null.

We seem to have preserved this logic when the `HomePage` was created. Now that the navbar logo navigates to `/`, I think it is safe to remove this. We can check in with product but I suspect we will fall in line with other products.